.travis.yml

```diff
@@ -10,8 +10,6 @@ before_install:
     - if [ "$TRAVIS_OS_NAME" = "osx" ]; then brew update ; brew outdated cmake || brew upgrade cmake ; fi
     - cmake --version
 
-before_script: travis_wait
```

Comment:
@aashish24 nope that was me - and it's redundant and did nothing.. For it to work to extend timeouts it needs to precede the command line arguments